src/tokenizer.rs

```diff
@@ -1259,7 +1283,7 @@ mod tests {
     fn tokenize_select_1() {
         let sql = String::from("SELECT 1");
         let dialect = GenericDialect {};
-        let mut tokenizer = Tokenizer::new(&dialect, &sql);
+        let mut tokenizer = Tokenizer::new(&dialect, &sql, &TokenizerOptions { no_escape: false });
```

Comment:
Could we make this look more like:

```suggestion
        let mut tokenizer = Tokenizer::new(TokenizerOptions::new(&dialect), &sql);
```

Reply:
Thanks!!!
I implemented and used `TokenizerOptions::default()` and `TokenizerOptions::with_no_escape(bool)`. :)

[29cb1e2](https://github.com/sqlparser-rs/sqlparser-rs/pull/870/commits/29cb1e20f618b45ea9e9d9418723f8800bcaef8d)